src/pages/FavoritesPage.js

```diff
@@ -9,22 +9,30 @@ const FavoritesPage = ({ favcharacterIDs, handleBookmarking }) => {
   const loadCharacters = () => {
   const basicUrl = 'https://rickandmortyapi.com/api/character/';
   
-  
-  fetch(basicUrl+favcharacterIDs)
+  if (favcharacterIDs.length>1) { 
```

Comment:
For better readability, I would recommend using spaces here:
```suggestion
  if (favcharacterIDs.length > 1) { 
```